### PR TITLE
network: set tcp congestion default

### PR DESCRIPTION
**Pull request by sduponch**

### CODE REVIEW

File: overthebox/defaults/network.defaults [hunk 1]
```diff
@@ -10,6 +10,14 @@ if [ $? -ne 0 ]; then
 EOF
 fi
 
+uci get network.globals.tcp_congestion_control >/dev/null 2>&1
```
Comment (gregdel) [inline overthebox/defaults/network.defaults]:
```shell
DEFAULT_TCP_CONGESTION_CONTROL="cubic"
tcp_congestion=$(uci get -q network.globals.tcp_congestion_control)
if [ "$tcp_congestion" != "$DEFAULT_TCP_CONGESTION_CONTROL" ]...
```

File: overthebox/defaults/network.defaults [hunk 2]
```diff
@@ -10,6 +10,14 @@ if [ $? -ne 0 ]; then
...
+if [ $? -ne 0 ]; then
+  uci -q batch <<-EOF >/dev/null
```
Comment (gregdel) [inline overthebox/defaults/network.defaults]: I don't see why we should use `batch`

File: overthebox/defaults/network.defaults [hunk 3]
```diff
@@ -10,6 +10,14 @@ if [ $? -ne 0 ]; then
...
+	set network.globals=globals
+	set network.globals.tcp_congestion_control='cubic'
```
Comment (gregdel) [inline overthebox/defaults/network.defaults]: `$DEFAULT_TCP_CONGESTION_CONTROL` instead of `cubic` here to if you agree with my first comment

Comment (angt) [inline overthebox/defaults/network.defaults]: nothing about this file ? https://github.com/ovh/overthebox-feeds/blob/master/overthebox/defaults/tcp_congestion_control.defaults

File: overthebox/defaults/network.defaults (hunk 1 above)
Comment (sduponch) [inline overthebox/defaults/network.defaults]: So we force the user to use only "cubic" ?
My main idea was to set "cubic" by default to override unexpected kernel default value like olia.

File: overthebox/defaults/network.defaults (hunk 2 above)
Comment (sduponch) [inline overthebox/defaults/network.defaults]: right

**Pull request closed without merging**